resources/lang/za/validation.php

```diff
@@ -0,0 +1,15 @@
+<?php
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+    'attached' => 'Hierdie: Attribuut is reeds aangeheg.'."\r\n".'Lokhu: imfanelo isivele ifakwe.',
```

Comment:
What is it?

Reply:
It's line separated translations in Zulu and Afrikaans. South Africa has 11 Official languages, aside from English those two are the most common.